modules/virt-removing-interface-from-nodes.adoc

```diff
@@ -30,19 +32,29 @@ spec:
     node-role.kubernetes.io/worker: "" <3>
   desiredState:
     interfaces:
-      - name: br1
-        type: linux-bridge
-        state: absent <4>
+    - name: br1
+      type: linux-bridge
+      state: absent <4>
+    - name: eth1
+      type: ethernet <5>
+      state: up <6>
+      ipv4:
+        dhcp: true <7>
+        enabled: true <8>
 ----
 <1> Name of the policy.
 <2> Optional: If you do not include the `nodeSelector` parameter, the policy applies to all nodes in the cluster.
 <3> This example uses the `node-role.kubernetes.io/worker: ""` node selector to select all worker nodes in the cluster.
 <4> Changing the state to `absent` removes the interface.
+<5> The type of interface. This example creates an Ethernet networking interface.
```

Comment:
I would add an extra explanation line (sorry, I don't know what the official term is) for the eth1 interface - probably for the "name" entry - which say something like:
The name of the interface that is going to be unattached from the bridge interface.

I think it is important in order to clarify that eth1 is not some arbitrary node interface, because in this bridge teardown policy there's no link between the bridge and its ports (as opposed to the setup policy above, where eth1 is explicitly declared as a subordinate port of the bridge).

Reply:
We call them 'callouts' :)
Great point. I've added callout with description:
"The name of the interface that is to be unattached from the bridge interface."